### ISSUE TITLE
Internationalization (i18n) component template/option

### ISSUE THREAD

**Issue by ChristianMurphy** [elmsln/issues#16]
Based off a discussion on Apereo Slack https://apereo.slack.com/archives/C02B68G62/p1549058151102500

There is interest in putting together an option and/or template for multi-language support within a web component.

**Comment by btopro** [comment]
#141 moving to this issue. Good discussion here but we have a specific one to explore
